Insights/InputEntity_Host/GroupAdditions.yaml

```diff
@@ -113,21 +113,21 @@ Insights:
   Type: LineChart
 
  AdditionalQuery:
-  Text: "See All Group Additions"
+  Text: "See all group additions"
   Query: "project TimeGenerated, EventID, Activity, Computer, MemberAdded, MemberName, MemberSid, TargetUserName, TargetDomainName, TargetSid, UserPrincipalName, SubjectUserName, SubjectUserSid, WellKnownGroupSID, WellKnownLocalSID, _ResourceId, SourceComputerId, timestamp, HostCustomEntity | order by TimeGenerated desc"
 
 Activities:
  EnabledByDefault: true
  Items:
    - Id: b880ad94-f905-4ba8-8a3f-9088b19b12fa
-     Description: Account added to Local Administrators Group
+     Description: Account added to Local Administrators group
```

Comment:
Checked online.  In the AD Docs, local administrators group is not capitalized
https://docs.microsoft.com/en-us/azure/active-directory/devices/assign-local-admin
"To manage a Windows device, you need to be a member of the **local administrators group**. "

Reply:
Done